samples/samples_test.go

```diff
@@ -63,6 +64,11 @@ func TestIntegration_Sample(t *testing.T) {
 	assertContains(t, out, "Fetched all concerts")
 	out = runSample(t, samples.PrintAlbumsReleaseBefore1900, db, "failed to fetch albums released before 1900")
 	assertContains(t, out, "was released at")
+
+	out = runSample(t, jsonsamples.UpdateDataWithJsonColumn, db, "failed to update data with json")
```

Comment:
This sample only shows how to work with an existing JSON column. We also have the file `migrator_emulator_test.go` that contains a `Venue` model. That model also has a `description` field of type `string`. Can we change that type to `spanner.NullJson` and ensure that our migrator then creates a JSON column? (And/or fix it if it doesn't)